pkg/cli/completion.go

```diff
@@ -11,17 +11,27 @@ func (r *Runner) newCompletionCommand() *cli.Command {
 	// https://cli.urfave.org/v2/#bash-completion
 	return &cli.Command{
 		Name:  "completion",
-		Usage: "Output shell completion script for bash or zsh",
-		Description: `Output shell completion script for bash or zsh
-Run these commands in .bash_profile or .zprofile
+		Usage: "Output shell completion script for bash, zsh, or fish",
+		Description: `Output shell completion script for bash, zsh, or fish
+Run these commands in .bash_profile, .zprofile, or ~/.config/config.fish.
 e.g.
 .bash_profile
 
-if command -v aqua &> /dev/null; then source <(aqua completion bash); fi
+if command -v aqua &> /dev/null; then
+	source <(aqua completion bash)
+fi
 
 .zprofile
 
-if command -v aqua &> /dev/null; then source <(aqua completion zsh); fi
+if command -v aqua &> /dev/null; then
+	source <(aqua completion zsh)
+fi
+
+~/.config/config.fish
+
+if command -v aqua &> /dev/null; 
+	aqua completion fish | source
+end
```

Comment:
FWIW, that does not seem to be the recommended way: https://fishshell.com/docs/current/completions.html#where-to-put-completions
(I used that command in the issue for brevity)

Rather, I would run

```bash
aqua completion fish > .config/fish/completions/aqua/fish
```

once after installation/upgrade. No additional hook is necessary.

Reply:
Thank you for your review.
Fixed. dee19a29b0349d7ff6b2af7c785bf5aaf5e093f9